src/pages/code-reviews-getting-started/index.md

```diff
@@ -0,0 +1,88 @@
+---
+title: "Code reviews: getting started"
+date: 2018-05-03T12:07:50+03:00
+draft: false
+---
+
+What if I told you there is this really straightforward technique to make your project run smoother, faster, and with fewer bugs 🤔. Better yet, it’s completely free, requires only minimal tooling, and any developer can learn to use it in a matter of minutes 😳. This is how good code reviews are, and why you should use them.
+
+_Author: Thibaud Colas, Software Architect at Codemate Ltd._
+
+---
+
+## Why it’s good
+
+Developers tend to think of productivity in terms of “how fast can I write the code for this new feature”. We also tend to say “I write code” when describing our jobs. In practice though, professional software development is a matter of understanding the code that is already there: we simply spend much more time reading than writing. Productivity thus depends more on the overall code’s quality and maintainability, rather than the skills of individual contributors making changes.
+
+Joel Spolsky puts it neatly:
+
+> [It’s harder to read code than to write it](https://www.joelonsoftware.com/2000/04/06/things-you-should-never-do-part-i/).
```

Comment:
This transition from Joel Spolsky to John Papa feels kinda quick. Before transitioning to the next guy, it would be beneficial to tell some anecdotes here why Joel Spolsky is right, or what was the article about.

I wrote some quick ideas down below. Please feel free to cherry pick pieces, or ditch all of it and use them as inspiration for something completely new!

> In the article above, Joel explains why he feels that rewriting big applications from scratch is never a good idea. He also goes into detail how to combat smelly codebases and what are alternatives to big rewrites.
>
> At the time of writing, Netscape had just shipped a public beta of their brand new major release. Since they decided to do a full rewrite which took them 3 years, their competitors were able to gain leverage and Netscape's market share declined. Joel argues that such big rewrites should never be done, and we should focus on code quality and gradual improvements instead. Although the article is now over 18 years old, it still holds true to this day.
>
> From our anecdotal experiences, if we pay attention to code quality right from the beginning, we'll thank ourselves later. If two persons or more understand and find a piece of code well written now, chances are that it will be more pleasant to maintain later.

Reply:
I quite like it as-is, but I can see why if you don't know those people that well it feels weird to name-drop people in that quick of a succession. And it's not fair to expect the audience of this post to be familiar with those names.

I'll reword to rely less on the names, but I won't add new content summarising either of those resources as I think it breaks the flow of this article.